package/MDAnalysis/analysis/encore/utils.py

```diff
@@ -0,0 +1,400 @@
+# utils.py --- Utility functions for encore package
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from multiprocessing.sharedctypes import SynchronizedArray
+from multiprocessing import Process, Manager
+from numpy import savez, load, zeros, array, float64, sqrt, atleast_2d, \
+    reshape, newaxis, zeros, dot, sum, exp
+import numpy as np
+import sys
+try:
```

Comment:
Thanks. Scipy dependencies have now been isolated to the relevant functions, and tests have been wrapped (4e1cee0e04cdb6a1e30303dbbaf26197f5373d95 and 349974cdd64bb4fd63e0690c786a248b277cdfb8)